docs/cloud-native-security/vuln-management-faq.asciidoc

```diff
@@ -11,6 +11,10 @@ The CNVM integration uses various security data sources. The complete list can b
 
 CNVM uses the open source scanner https://github.com/aquasecurity/trivy[Trivy] v0.35.
 
+*What system architectures are supported?*
+
+Because of Trivy's limitations, CNVM works only on ARM. Other architectures are not supported.
```

Comment:
Clarifying with engineering on https://github.com/elastic/cloudbeat/issues/731#issuecomment-2181156246
I think we need to be clear its about the CNVM agent running on ARM based image and not about scanning only ARM based images.